ReadMe.md

```diff
@@ -25,19 +25,20 @@ require (
 )
 ```
```

Comment:
emm. When user read the doc, if he use go mod, he even won't read `Package replace` chapter, would you add the    
replace notice to `use go mod` chapter too.  And `Package replace` is equative to `Use go mod`, a bit weird.